docs/diagnostics/PX1007.md

```diff
@@ -47,9 +54,11 @@ The diagnostic is not displayed for the following items:
    - `LastModifiedByScreenID`
    - `LastModifiedDateTime`
    - `TStamp`
-   - `NoteID`.
+   - `NoteID`
+   - `Attributes`
+   - `GroupMask`
 
-If you suppress this diagnostic with a comment, the default comment message is the following: _Acuminator disable once PX1007 NoXmlCommentForPublicEntityOrDacProperty to be documented later_.
+To remove a DAC or its property from documentation, instead of suppressing the diagnostic, add the `exclude` tag.
```

Comment:
Need to mention DAC extensions too:
```suggestion
To remove a DAC, DAC extension or its property from documentation, instead of suppressing the diagnostic, add the `exclude` tag.
```